data/dataset.py

```diff
@@ -71,7 +196,11 @@ def __getitem__(self, idx: int) -> Union[Data, HeteroData]:
         user_features = self.graph[Constants.node_user].x[idx]
 
         # Prepare connected article features
-        article_features = self.graph[Constants.node_item].x[all_touched_edges]
+        article_features = self.graph[Constants.node_item].x[
+            torch.sort(all_touched_edges)[0]
+        ]
+        print("FASDFSDFAD")
```

Comment:
probably not needed

Reply:
yes
Also took out a bunch of superflous remapping in first_user